tests/unit/src/test/scala/tests/testProvider/TestSuitesProviderSuite.scala

```diff
@@ -45,7 +45,9 @@ class TestSuitesProviderSuite extends BaseLspSuite("testSuitesFinderSuite") {
           |        "fullyQualifiedName": "NoPackage",
           |        "className": "NoPackage",
           |        "location": {
-          |          "uri": "${classUriFor("app/src/main/scala/NoPackage.scala")}",
+          |          "uri": "${classUriFor(
+        "app/src/main/scala/NoPackage.scala"
+      )}",
```

Comment:
These are the cases that I really don't like. I wish it didn't split here. However, I see further down in the file it does the same thing.

Reply:
could you try to add the following to config?
```
newlines.inInterpolation = avoid
```
It might help with this case

Reply:
huh, I didn't even see `avoid` in the docs, thanks. I ended up not adding this since it won't "force" all the other places this is done, so we'd need to manually go through and change those which would create an even bigger diff than these two new cases. So because the other ones are already doing this I just left it.